### PR TITLE
Delegating-edits

### PR DESCRIPTION
**Pull request by idalithb**
-edits for readability

### CODE REVIEW

File: website/pages/en/network/delegating.mdx [hunk 1]
```diff
@@ -2,13 +2,23 @@
...
+- They earn a portion of an Indexer's query fees and rewards by delegating to them.
+
+**How does this work?**
+
+- The number of queries an Indexer can process depends on their own stake, **the delegated stake**, and the price the Indexer charges for each query. Therefore, the more stake allocated to an Indexer, the more potential queries an Indexer can process.
 
 ## Delegator Guide
 
-This guide will explain how to be an effective Delegator in the Graph Network. Delegators share the earnings of the protocol alongside all Indexers based on their delegated stake. A Delegator must use their best judgment to choose Indexers based on multiple factors. Please note this guide will not cover steps such as setting up Metamask properly, as that information is widely available on the internet.
+Learn how to be an effective Delegator in the Graph Network.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:17]:
```suggestion
Learn how to be an effective Delegator in The Graph Network.
```

File: website/pages/en/network/delegating.mdx [hunk 2]
```diff
@@ -2,13 +2,23 @@
...
+
+Delegators share the earnings of the protocol alongside all Indexers based on their delegated stake. Therefore, they must use their best judgment to choose Indexers based on multiple factors.
+
+> Please note this guide does not cover steps such as setting up Metamask properly, as that information is widely available online.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:21]:
```suggestion
> Please note this guide does not cover steps such as setting up MetaMask properly, as that information is widely available online.
```

File: website/pages/en/network/delegating.mdx [hunk 3]
```diff
@@ -24,15 +34,19 @@ Listed below are the main risks of being a Delegator in the protocol.
...
+- You will be charged 0.5% every time you delegate. This means that if you delegate 1000 GRT, you will automatically burn 5 GRT.
+
+- In order to be safe, you should calculate your potential return when delegating to an Indexer. For example, you might calculate how many days it will take before you have earned back the 0.5% tax on your delegation.
 
 ### The delegation unbonding period
 
 Whenever a Delegator wants to undelegate, their tokens are subject to a 28-day unbonding period. This means they cannot transfer their tokens or earn any rewards for 28 days.
 
-Another thing to consider is how to choose an Indexer wisely. If you choose an Indexer that is not trustworthy or not doing a good job, you will want to undelegate. This means you will be losing a lot of opportunities to earn rewards, which can be just as bad as burning GRT.
+**Why is this important?**
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:47]: Heading?

File: website/pages/en/network/delegating.mdx [hunk 4]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 ### Choosing a trustworthy Indexer with a fair reward payout for Delegators
 
-This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
+In order to understand how to choose a trustworthy Indexer, you need to understand the Delegation Parameters.
+
+#### Delegation Parameters
 
-Indexing Reward Cut - The indexing reward cut is the portion of the rewards that the Indexer will keep for themselves. This means that if an Indexer's rewards are set to 100%, as a Delegator you will get 0 indexing rewards. If you see it set at 80% in the UI, that means as a Delegator, you will receive 20%. An important note - at the beginning of the network, Indexing Rewards will account for the majority of the rewards.
+- **Indexing Reward Cut** - The portion of the rewards the Indexer will keep for themselves.
+  - If an Indexer's rewards are set to 100%, as a Delegator, you will get 0 indexing rewards.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:63]:
```suggestion
  - If an Indexer's reward cut is set to 100%, as a Delegator, you will get 0 indexing rewards.
```

File: website/pages/en/network/delegating.mdx [hunk 5]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+  - If you see it set at 80% , as a Delegator, you will receive 20%.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:64]:
```suggestion
  - If it is set to 80% , as a Delegator, you will receive 20%.
```

File: website/pages/en/network/delegating.mdx [hunk 6]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+    > Note: At the beginning of the network, Indexing Rewards will account for the majority of the rewards.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:65]: "At the beginning of the network"? I think this whole note should be removed. 😛

File: website/pages/en/network/delegating.mdx [hunk 7]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
 <div sx={{ mx: 'auto', maxWidth: '700px' }}>
   ![Indexing Reward Cut](/img/Indexing-Reward-Cut.png) *The top Indexer is giving Delegators 90% of the rewards. The
   middle one is giving Delegators 20%. The bottom one is giving Delegators ~83%.*
 </div>
 
-- Query Fee Cut - This works exactly like the Indexing Reward Cut. However, this applies explicitly to returns on the query fees the Indexer collects. It should be noted that at the start of the network, returns from query fees will be very small compared to the indexing reward. It is recommended that you pay attention to the network to determine when the query fees in the network will start to be more significant.
+- **Query Fee Cut** - This is the same as the Indexing Reward Cut, but it specifically applies to returns on the query fees the Indexer collects.
+
+  > Note: At the start of the network, returns from query fees will be very small compared to the indexing reward. It is recommended that you pay attention to the network to determine when the query fees will start to become more significant.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:74]: I would also remove this note, or at least rephrase it to not mention "at the start of the network", since the network has been up for 3 years.

File: website/pages/en/network/delegating.mdx [hunk 8]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
 ![Delegation Image 3](/img/Delegation-Reward-Formula.png)
 
 ### Considering the Indexer's delegation pool
 
-Another thing a Delegator has to consider is what proportion of the Delegation Pool they own. All delegation rewards are shared evenly, with a simple rebalancing of the pool determined by the amount the Delegator has deposited into the pool. This gives the Delegator a share of the pool:
+Delegators should consider the proportion of the Delegation Pool they own.
 
-![Share formula](/img/Share-Forumla.png)
+- All delegation rewards are shared evenly, with a pool rebalancing based on the amount the Delegator deposited into the pool
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:103]: Note sure why you used a list with a single item here, instead of a paragraph?

File: website/pages/en/network/delegating.mdx [hunk 9]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
-Using this formula, we can see that it is possible for an Indexer offering only 20% to Delegators to actually provide a better reward than an Indexer giving 90%.
+This gives the Delegator a share of the pool:
 
-Therefore, a Delegator can do the math to determine that the Indexer offering 20% to Delegators is offering a better return.
+![Share formula](/img/Share-Forumla.png)
+
+> The formula above shows that it is possible for an Indexer offering only 20% to Delegators can provide a better reward than an Indexer giving 90%. Simply do the math to determine the best reward.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:109]:
```suggestion
> The formula above shows that it is possible for an Indexer offering only 20% to Delegators to provide a better reward than an Indexer giving 90%. Simply do the math to determine the best reward.
```

File: website/pages/en/network/delegating.mdx [hunk 10]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
 ### Considering the delegation capacity
 
-Another thing to consider is the delegation capacity. Currently, the Delegation Ratio is set to 16. This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of Delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
+Finally, consider the delegation capacity.
+
+- Currently, the Delegation Ratio is set to 16.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:115]: Why is this a list?

File: website/pages/en/network/delegating.mdx [hunk 11]
```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
-Imagine an Indexer with 100,000,000 GRT delegated to them, but their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. So, all the Delegators and the Indexer, are earning way less rewards than they could be.
+**Why does this matter?**
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:117]: Heading?

File: website/pages/en/network/delegating.mdx (hunk 10 above)
Comment (benface) [reply website/pages/en/network/delegating.mdx:115]: Ah it's explained below, nevermind that second comment haha.

File: website/pages/en/network/delegating.mdx [hunk 12]
```diff
@@ -2,13 +2,23 @@
...
+Delegators are network participants who delegate (i.e., "stake") GRT to one or more Indexers.
 
-Delegators earn a portion of an Indexer's query fees and rewards by delegating to them. The amount of queries an Indexer can process depends on their own stake, the delegated stake, and the price the Indexer charges for each query. Therefore, the more stake that is allocated to an Indexer, the more potential queries they can process.
+- They help secure the network without running a Graph Node themselves.
+
+- They earn a portion of an Indexer's query fees and rewards by delegating to them.
+
+## How does this work?
+
+- The number of queries an Indexer can process depends on their own stake, **the delegated stake**, and the price the Indexer charges for each query. Therefore, the more stake allocated to an Indexer, the more potential queries an Indexer can process.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:13]: Sorry if I'm too picky about this, but not sure why we ever have lists with a single item. Shouldn't this be a paragraph?

Comment (idalithb) [reply website/pages/en/network/delegating.mdx:13]: No problem! Bullet points make it easier for UI

File: website/pages/en/network/delegating.mdx [hunk 13]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 ### Choosing a trustworthy Indexer with a fair reward payout for Delegators
 
-This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
+In order to understand how to choose a trustworthy Indexer, you need to understand the Delegation Parameters.
 
-Indexing Reward Cut - The indexing reward cut is the portion of the rewards that the Indexer will keep for themselves. This means that if an Indexer's rewards are set to 100%, as a Delegator you will get 0 indexing rewards. If you see it set at 80% in the UI, that means as a Delegator, you will receive 20%. An important note - at the beginning of the network, Indexing Rewards will account for the majority of the rewards.
+#### Delegation Parameters
+
+- **Indexing Reward Cut** - The portion of the rewards the Indexer will keep for themselves.
+  - If an Indexer's reward cut is set to 100%, as a Delegator, you will get 0 indexing rewards.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:63]:
```suggestion
  - If an Indexer's reward cut is set to 100%, as a Delegator, you will not get any indexing rewards.
```

File: website/pages/en/network/delegating.mdx (hunk 12 above)
Comment (idalithb) [reply website/pages/en/network/delegating.mdx]: -fine changing it for this 👍

Comment (benface) [reply website/pages/en/network/delegating.mdx]: I'm curious to have @0xa3k5's thoughts on this.

File: website/pages/en/network/delegating.mdx (hunk 13 above)
Comment (idalithb) [reply website/pages/en/network/delegating.mdx:63]: -I like the numbers here--make it more direct

File: website/pages/en/network/delegating.mdx [hunk 14]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+
+As you can see, in order to choose the right Indexer, you must consider multiple things.
 
-As you can see, in order to choose the right Indexer, you must consider multiple things. This is why we highly recommend exploring [The Graph Discord](https://discord.gg/graphprotocol) to determine which Indexers have the best social and technical reputations and which reward Delegators consistently. Many of the Indexers are very active in Discord and will be happy to answer your questions. Many of them have been Indexing for months on the testnet, and they are doing their best to help Delegators earn a good return, as it improves the health and success of the network.
+- It is highly recommend that you explore [The Graph Discord](https://discord.gg/graphprotocol) to determine which Indexers have the best social and technical reputations and which reward Delegators consistently.
+- Many Indexers are very active in Discord and will be happy to answer your questions.
+- Many of them have been Indexing for month, and they are doing their best to help Delegators earn a good return, as it improves the health and success of the network.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:77]:
```suggestion
- Many of them have been Indexing for months, and they are doing their best to help Delegators earn a good return, as it improves the health and success of the network.
```

Comment (idalithb) [reply website/pages/en/network/delegating.mdx:77]: great catch!

File: website/pages/en/network/delegating.mdx [hunk 15]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
 ![Delegation Image 3](/img/Delegation-Reward-Formula.png)
 
 ### Considering the Indexer's delegation pool
 
-Another thing a Delegator has to consider is what proportion of the Delegation Pool they own. All delegation rewards are shared evenly, with a simple rebalancing of the pool determined by the amount the Delegator has deposited into the pool. This gives the Delegator a share of the pool:
+Delegators should consider the proportion of the Delegation Pool they own.
 
-![Share formula](/img/Share-Forumla.png)
+All delegation rewards are shared evenly, with a pool rebalancing based on the amount the Delegator deposited into the pool
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:100]:
```suggestion
All delegation rewards are shared evenly, with a pool rebalancing based on the amount the Delegator deposited into the pool.
```

File: website/pages/en/network/delegating.mdx [hunk 16]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+![Share formula](/img/Share-Forumla.png)
+
+> The formula above shows that it is possible for an Indexer offering only 20% to Delegators to provide a better reward than an Indexer giving 90%. Simply do the math to determine the best reward.
 
 ### Considering the delegation capacity
 
-Another thing to consider is the delegation capacity. Currently, the Delegation Ratio is set to 16. This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of Delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
+Finally, consider the delegation capacity.
+
+Currently, the Delegation Ratio is set to 16.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:112]: These paragraphs are so short haha, maybe better to merge them?

```suggestion
Finally, consider the delegation capacity. Currently, the Delegation Ratio is set to 16.
```

File: website/pages/en/network/delegating.mdx [hunk 17]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
-Imagine an Indexer with 100,000,000 GRT delegated to them, but their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. So, all the Delegators and the Indexer, are earning way less rewards than they could be.
+### Why does this matter?
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:114]:
```suggestion
#### Why does this matter?
```

File: website/pages/en/network/delegating.mdx [hunk 18]
```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+Imagine an Indexer with 100,000,000 GRT delegated to them, but their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. So, both the Delegators and the Indexers are earning less rewards than they could be.
 
 Therefore, a Delegator should always consider the Delegation Capacity of an Indexer, and factor it into their decision making.
 
 ## Delegator FAQs and Bugs
 
 ### MetaMask "Pending Transaction" Bug
 
-**When I try to delegate my transaction in MetaMask, it appears as "Pending" or "Queued" for longer than expected. What should I do?**
+1. When I try to delegate my transaction in MetaMask, it appears as "Pending" or "Queued" for longer than expected. What should I do?\*\*
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:126]:
```suggestion
1. When I try to delegate my transaction in MetaMask, it appears as "Pending" or "Queued" for longer than expected. What should I do?
```
(or was there a reason for these asterisks?)

File: website/pages/en/network/delegating.mdx [hunk 19]
```diff
@@ -24,15 +34,19 @@ Listed below are the main risks of being a Delegator in the protocol.
 
 Delegators cannot be slashed for bad behavior, but there is a tax on Delegators to disincentivize poor decision-making that could harm the integrity of the network.
 
-It is important to understand that every time you delegate, you will be charged 0.5%. This means that if you are delegating 1000 GRT, you will automatically burn 5 GRT.
+As a Delegator, it's important to understand the following:
 
-In order to be safe, a Delegator should calculate their potential return when delegating to an Indexer. For example, a Delegator might calculate how many days it will take before they have earned back the 0.5% tax on their delegation.
+- You will be charged 0.5% every time you delegate. This means that if you delegate 1000 GRT, you will automatically burn 5 GRT.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:39]:
```suggestion
- You will be charged 0.5% every time you delegate. This means that if you delegate 1,000 GRT, you will automatically burn 5 GRT.
```

File: website/pages/en/network/delegating.mdx [hunk 20]
```diff
@@ -41,64 +55,87 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
-This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
+In order to understand how to choose a trustworthy Indexer, you need to understand the Delegation Parameters.
 
-Indexing Reward Cut - The indexing reward cut is the portion of the rewards that the Indexer will keep for themselves. This means that if an Indexer's rewards are set to 100%, as a Delegator you will get 0 indexing rewards. If you see it set at 80% in the UI, that means as a Delegator, you will receive 20%. An important note - at the beginning of the network, Indexing Rewards will account for the majority of the rewards.
+#### Delegation Parameters
+
+- **Indexing Reward Cut** - The portion of the rewards the Indexer will keep for themselves.
+  - If an Indexer's reward cut is set to 100%, as a Delegator, you will get 0 indexing rewards.
+  - If it is set to 80% , as a Delegator, you will receive 20%.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:64]:
```suggestion
  - If it is set to 80%, as a Delegator, you will receive 20%.
```

File: website/pages/en/network/delegating.mdx [hunk 21]
```diff
@@ -41,64 +55,87 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
+- **Query Fee Cut** - This is just like the Indexing Reward Cut, but it applies to returns on the query fees that the Indexer collects.
+
+As you can see, in order to choose the right Indexer, you must consider multiple things.
 
-As you can see, in order to choose the right Indexer, you must consider multiple things. This is why we highly recommend exploring [The Graph Discord](https://discord.gg/graphprotocol) to determine which Indexers have the best social and technical reputations and which reward Delegators consistently. Many of the Indexers are very active in Discord and will be happy to answer your questions. Many of them have been Indexing for months on the testnet, and they are doing their best to help Delegators earn a good return, as it improves the health and success of the network.
+- It is highly recommend that you explore [The Graph Discord](https://discord.gg/graphprotocol) to determine which Indexers have the best social and technical reputations and which reward Delegators consistently.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:75]: Just a suggestion, not sure if it's objectively better but reads better to me:
```suggestion
- It is highly recommend that you explore [The Graph Discord](https://discord.gg/graphprotocol) to determine which Indexers have the best social and technical reputations and which ones reward Delegators consistently.
```

File: website/pages/en/network/delegating.mdx [hunk 22]
```diff
@@ -41,64 +55,87 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
...
 
 ### Considering the delegation capacity
 
-Another thing to consider is the delegation capacity. Currently, the Delegation Ratio is set to 16. This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of Delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
+Finally, consider the delegation capacity. Currently, the Delegation Ratio is set to 16.
 
-Imagine an Indexer with 100,000,000 GRT delegated to them, but their capacity is only 16,000,000 GRT. This means effectively, 84,000,000 GRT tokens are not being used to earn tokens. So, all the Delegators and the Indexer, are earning way less rewards than they could be.
+#### Why does this matter?
+
+This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of Delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
```
Comment (benface) [inline website/pages/en/network/delegating.mdx:114]:
```suggestion
This means that if an Indexer has staked 1,000,000 GRT, their Delegation Capacity is 16,000,000 GRT of delegated tokens that they can use in the protocol. Any delegated tokens over this amount will dilute all the Delegator rewards.
```

**Review by benface — APPROVED** [review]
Great work!
